website/docs/reference/widgets/input.md

```diff
@@ -1,258 +1,118 @@
-
-
-#### Error message
-
-If a user enters an incorrect value, the input widget shows a message "invalid input." You can change this message by using the `Error message` property to provide better feedback on the input given by the user.
-
-<VideoEmbed host="youtube" videoId="oeUHJhM4zyU" title="Error Message" caption="Error Message"/>
-
-#### Required
-
-Entering a value in the input box is mandatory when the required property is enabled. You can also write a JS code to link this property to a user action. Click on `JS` next to the `Required` to write JavaScript code.
-
-For example, drag a checkbox widget `checkbox1` onto the canvas and bind it to the `Required` property. To enable the `Required` when the user checks the checkbox, add the following JavaScript code in the Required property:
+As an example, imagine that you are creating rules for a "Create Password" field in an account registration form. If you want to be sure that new passwords don't contain certain strings, you may write:
 
+```javascript
+// Valid property of an Input widget
+{{
+  !["password", "123", "admin"].some(subStr => {
+    return PasswordInput.text.toLowerCase().includes(subStr)
+  })
+}}
 ```
-{{Checkbox1.isChecked}}
-```
-
-When you tick the checkbox, it enables the Required property, and the input box shows an error message "This field is required" if you haven't entered any input.
-
-<VideoEmbed host="youtube" videoId="2hqT02HCah8" title="Required" caption="Required"/>
-
-#### Show step arrows
-
-:::note
-The **Show step arrows** property is only available for the number data type.
-:::
-
-When `Show step arrows` property is turned on, the step arrows are displayed and user can use them to increase or decrease the value inside the widget. You can also use JavaScript code to dynamically show or hide the step arrows based on certain conditions or events. This can be useful if you want to enable or disable the step arrows based on the current state of the widget or other factors.
-
-<VideoEmbed host="youtube" videoId="tctkh2aZWP8" title="Show step arrows" caption="Show step arrows"/>
-
-
-#### Visible
-
-`Visible` controls the widget's visibility on the app's page. The widget is visible on the published app if you turn off this property. You can also write a JS code that links the Visible property to a user action. Click on `JS` next to the `Visible` to write JavaScript code.
 
-For example, drag a checkbox widget `checkbox1` onto the canvas and bind it to the `Visible` property. To enable the `Visible` when the user checks the checkbox, add the following JavaScript code:
+The **Regex** property is used to compare input to a regular expression that you have defined. If the user's input doesn't match the pattern, it's considered invalid. This is important when you need to be sure that the content exactly follows the formatting of your datasource.
 
```

Comment:
After validation properties, we should have a section for using Input widget to search data in a list which does not have its own inbuilt search

Reply:
Should this be a how-to guide or sample app? Maybe it's more common than I think -- but this topic sounds like it might be more of a possible application of the widget rather than the primary intent of the doc